fees/lyra-v2.ts

```diff
@@ -0,0 +1,27 @@
+import { Adapter } from "../adapters/types";
+import { CHAIN } from "../helpers/chains";
+import axios from "axios";
+
+async function getLyraV2Fees({endTimestamp}) {
+  const data = await axios.get(`https://api.lyra.finance/public/statistics?instrument_name=ALL&end_time=${endTimestamp*1e3}`, {
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  return {
+    dailyFees: data.data.result.daily_fees.toString(),
```

Comment:
what does `data.data.result.daily_fees` represent? Is it midnight to midnight data?